### PR TITLE
Fixing unclosed tags warnings on Scaladoc for #4996

### PR DESCRIPTION
**Pull request by mtrovo**
Fixing all the unclosed tags warnings on Scaladoc.

### CODE REVIEW

File: framework/src/play/src/main/scala/play/api/Configuration.scala [hunk 1]
```diff
@@ -269,7 +269,7 @@ case class Configuration(underlying: Config) {
    * }}}
    *
    * A configuration error will be thrown if the configuration value is not a valid `Boolean`.
-   * Authorized vales are `yes/no or true/false.
+   * Authorized vales are `yes/no` or `true/false`.
```
Comment (BattleBeaver) [inline framework/src/play/src/main/scala/play/api/Configuration.scala]: Strange you did not remove ` sign here.

Comment (mtrovo) [inline framework/src/play/src/main/scala/play/api/Configuration.scala]: I thought the original intent was to display the yes/no as monospaced.
Do you think it's better to remove it all?

**Comment by marcospereira** [comment]
See #5203 too. There is some overlapping between these two pull requests.

**Comment by nafg** [comment]
The / should not be monospaced. Also while you're at it, values not vales.
;)

**Comment by mtrovo** [comment]
@nafg just fixed it, mind taking a look again?
@marcospereira I fixed all unclosed tags warnings without checking other PRs and I think just one file is overlapping, should I drop the file from my commit?

**Comment by marcospereira** [comment]
@mtrovo right now you don't need worry about it. If we decide to merge your PR first, then a rebase will be required in the other PR and vice versa. To me, looks like this PR is ready to be merged. I'm typing from my cellphone now and will take a closer look later tonight. ;-)

**Comment by marcospereira** [comment]
@mtrovo thank you so much!

**Pull request merged**